### ISSUE TITLE
Correct formatting for "Sensibo Out" node.

### ISSUE THREAD

**Issue by adroxx** [thetonto/node-red-contrib-sensibo#10]
Hey all, 

Having some issues with the "Sensibo Out" node. I have the following in my msg.payload which I'm placing in an "Inject" node right before the "Sensibo Out" node, but it doesn't seem to work. What am I missing?

{
    "acState": {
        "on": true,
        "mode": "cool",
        "fanLevel": "quiet",
        "targetTemperature": 22,
        "temperatureUnit": "C",
        "swing": "stopped"
    }
}

**Comment by thetonto** [comment]
Hi adroxx
May I suggest that you trim down the message doing to the plugin and then find out what item is not working.

Been a while since I updated this, and I left if quite basic.  The help instructions show the messages that go thru and now thinking on it the termp unit was not one I built into the process.  Try taking that out and let me know.

**Comment by adroxx** [comment]
Hey bud, thanks for creating the plugin, and for the reply.

I have tried a number of variations on the JSON. I just tried the following, again, just to make sure, but it still doesn't work.

```
{
    "acState": {
        "on": true
    }
}
```

I also tried removing the acState.

```
{
        "on": true
 }
```

It's definitely calling the AirCon, as I hear the beep, and it returns the current values, but will not set new ones.

Any other ideas?

**Comment by adroxx** [comment]
It's not a huge deal, as I can easily accomplish just about anything via a 'change' and 'http request node', but ideally, I would like to use your plugin to make it a little cleaner.

**Comment by vlogicau** [comment]
Hey adroxx,
If you set the on value to "true" or "false" (i.e. string rather than binary) then it should fix your issue. I was having the same challenge and fixed it by changing the datatype.

**Comment by thetonto** [comment]
All this issue has now been fixed from v0.5.0 onwards.  The update already removes deprecated requests library
